src/odemis/util/synthetic.py

```
@@ -128,6 +128,42 @@ def psf_gaussian(
+    :param image: optional 2D array to use directly when shape indicates a 2D image (y > 1)
+    :param dtype: the data type of the output array (default is numpy.uint16)
+    :return: a numpy array containing the simulated peak, either as a 1D Gaussian or directly from the provided image
+    """
+
+    # Standardize shape to a tuple, so we can index it
+    if isinstance(shape, (int, numpy.integer)):
+        shape = (shape,)
+
+    # Determine resolution.y (height)
+    # If it's a 1D shape, resolution_y is 1
+    resolution_y = shape[0] if len(shape) > 1 else 1
```

Ruff (0.15.5)

[error] Local variable `resolution_y` is assigned to but never used

Remove assignment to unused variable `resolution_y`

(F841)